plugin.video.vrt.nu/resources/lib/vrtplayer/streamservice.py

```diff
@@ -1,66 +1,116 @@
 from abc import ABCMeta, abstractmethod
 import requests
+import urllib
+import json
+import os
+import datetime
+import time
+import _strptime
+import xbmc
 from resources.lib.helperobjects import helperobjects
 
 class StreamService:
```

Comment:
Now we only have one streamservice => UrlToStreamService class maybe you can remove the base StreamService class? :)